src/gui/torrentcontenttreeview.cpp

```diff
@@ -107,6 +108,78 @@ void TorrentContentTreeView::keyPressEvent(QKeyEvent *event)
     }
 }
 
+void TorrentContentTreeView::setupDownloadPriorityMenu(QMenu *menu, const bool createSubMenu) const
+{
+    const auto model = qobject_cast<TorrentContentFilterModel *>(this->model());
+    Q_ASSERT(model);
+
+    const auto getSelectedRows = [this]() { return selectionModel()->selectedRows(TorrentContentModelItem::COL_PRIO); };
+
+    const auto applyPriorities = [&](const BitTorrent::DownloadPriority priority)
+    {
+        return [model, getSelectedRows, priority]()
+        {
+            for (const QModelIndex &index : asConst(getSelectedRows()))
+            {
+                model->setData(index, static_cast<int>(priority));
+            }
+            emit model->filteredFilesChanged();
+        };
+    };
+    const auto applyPrioritiesByOrder = [model, getSelectedRows]()
+    {
+        // Equally distribute the selected items into groups and for each group assign
+        // a download priority that will apply to each item. The number of groups depends on how
+        // many "download priority" are available to be assigned
+
+        const QModelIndexList selectedRows = getSelectedRows();
+        const qsizetype priorityGroups = 3;
+        const auto priorityGroupSize = std::max<qsizetype>(selectedRows.length() / priorityGroups, 1);
```

Comment:
instead of copy and paste the existing code, you decided to break the coding style.
`(selectedRows.length() / priorityGroups)`

Reply:
Fixed by 84aa302f83e1fd6dddb673cd983ac8384fdbbde3